lib/product.rb

```diff
@@ -0,0 +1,26 @@
+class Product
+  attr_reader :category, :name, :unit_price, :quantity
+
+  def initialize(category, name, unit_price, quantity)
+    @category = category
+    @name = name
+    @unit_price = unit_price
+    @quantity = quantity.to_i
+  end
+
+  def total_price
+    @unit_price * @quantity
+  end
+
+  def is_hoarded?
+    if @quantity > 10
```

Comment:
you just need this quantity > 10 in this method since it will automatically return true or false.. no need for the if/else statement.